modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/SyncManagerImpl.kt

```diff
@@ -577,6 +583,7 @@ class SyncManagerImpl @Inject constructor(
         )
         isLoggedInObservable.accept(true)
 
+        settings.setFullySignedOut(false)
```

Comment:
This wasn't being set when you created an account so the next time my refresh token failed I wasn't being signed out.